### YamlMime:JavaMember
uid: "com.azure.resourcemanager.datamigration.models.ValidateMigrationInputSqlServerSqlMITaskOutput.existingBackupErrors*"
fullName: "com.azure.resourcemanager.datamigration.models.ValidateMigrationInputSqlServerSqlMITaskOutput.existingBackupErrors"
name: "existingBackupErrors"
nameWithType: "ValidateMigrationInputSqlServerSqlMITaskOutput.existingBackupErrors"
members:
- uid: "com.azure.resourcemanager.datamigration.models.ValidateMigrationInputSqlServerSqlMITaskOutput.existingBackupErrors()"
  fullName: "com.azure.resourcemanager.datamigration.models.ValidateMigrationInputSqlServerSqlMITaskOutput.existingBackupErrors()"
  name: "existingBackupErrors()"
  nameWithType: "ValidateMigrationInputSqlServerSqlMITaskOutput.existingBackupErrors()"
  summary: "Get the existingBackupErrors property: Errors associated with existing backup files."
  syntax: "public List<ReportableException> existingBackupErrors()"
  returns:
    description: "the existingBackupErrors value."
    type: "<xref href=\"java.util.List?alt=java.util.List&text=List\" data-throw-if-not-resolved=\"False\" />&lt;<xref href=\"com.azure.resourcemanager.datamigration.models.ReportableException?alt=com.azure.resourcemanager.datamigration.models.ReportableException&text=ReportableException\" data-throw-if-not-resolved=\"False\" />&gt;"
type: "method"
metadata: {}
package: "com.azure.resourcemanager.datamigration.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-datamigration:1.0.0-beta.1
